lpc55_sign/src/bin/lpc55_sign.rs

```diff
@@ -137,15 +157,293 @@ fn main() -> Result<()> {
                 &dest_bin, &dest_cmpa
             );
         }
-        ImageType::EccImage {
+        Command::EccImage {
             src_bin,
             priv_key,
             dest_bin,
         } => {
             sign_ecc::ecc_sign_image(&src_bin, &priv_key, &dest_bin)?;
             println!("Done! ECC image written to {:?}", &dest_bin);
         }
+        Command::VerifySignedImage {
+            src_cmpa,
+            src_cfpa,
+            src_img,
+            verbose,
+        } => {
+            println!("=== CMPA ====");
+            let cmpa = {
+                let mut cmpa_bytes = [0u8; 512];
+                let mut cmpa_file = std::fs::File::open(src_cmpa)?;
```

Comment:
On any line that opens a file, I recommend adding an `anyhow` `context` modifier so that the error is more useful than what these tools currently print, which is just "no such file or directory" or some such -- given that this command takes three paths, it would be lovely to know _which one it's talking about!_

e.g. in this case, `.with_context(|| format!("opening {}", src_cmpa.display()))?`

Reply:
Good idea, fixed